app/FreshRSS.php

```diff
@@ -111,7 +111,12 @@ private static function loadNotifications() {
 	public static function preLayout() {
 		switch (Minz_Request::controllerName()) {
 			case 'index':
-				header("Content-Security-Policy: default-src 'self'; child-src *; frame-src *; img-src * data:; media-src *");
+				$urlToAuthorize = array_filter(array_map(function($a) {
+					if ('POST' === $a['method']) {
+						return $a['url'];
+					}
+				}, FreshRSS_Context::$user_conf->sharing));
+				header(sprintf("Content-Security-Policy: default-src 'self' %s; child-src *; frame-src *; img-src * data:; media-src *", implode(' ', $urlToAuthorize)));
```

Comment:
It is not a big deal, but it would be better to keep `default-src 'self'` and update only what you need, which is likely `connect-src`. Otherwise, `default-src` also applies to scripts, css, etc.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/connect-src

Reply:
I wasn't aware of that. I'll modify accordingly.